src/js/embed.js

```diff
@@ -63,89 +45,70 @@ function setupVisibilityMonitoring() {
                 console.log('%c NOT VISIBLE', 'background: #222; color: #bada55');
             }
         }
-    });    
-}
-
-var formats = {
-    flat: {
-        preprocess: (data) => data,
-        template: questionAndAnswer
-    },
-    expandable: {
-        preprocess: (data) => {
-            // TODO: don't modify data in place!
-            let words = data.Answer.split(' ');
-            data.mainAnswer = words.slice(0, 65).join(' ');
-            data.extraAnswer = words.slice(65).join(' ');
-            return data;
-        },
-        template: expandableQuestionAndAnswer
-    },
-    carousel: {
-        preprocess: (data) => data,
-        template: ''
-    }
+    });
 }
 
-window.init = function init(el, config) {
-
-    iframeMessenger.enableAutoResize();
-
-    setupVisibilityMonitoring();
+const getQueryParams = () => {
```

Comment:
what's the advantage of this over `function getQueryParams()`? to stop you accidentally defining the same function twice?